app/furniture/marketplace/management_component.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="flex flex-wrap items-baseline">
+  <h2 class="ml-2 mt-1">
+    <%= t('marketplace.marketplace.manage_header') %>
+  </h2>
+  <p class="ml-2 mt-1 truncate text-sm text-gray-500">
+    <%= marketplace.room.name %>
+  </p>
+</div>
+<%= render CardComponent.new(classes: "flex mt-3 px-0") do %>
+  <div class="flex md:w-1/5 flex-col">
+    <nav class="flex grow flex-col gap-y-5 overflow-y-auto border-r border-gray-200 p-2 md:p-5">
+      <ul role="list" class="list-none pl-0">
+
+        <%- if policy(marketplace).edit? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(location: marketplace.location(:edit)) do |menu_item| %>
+            <% menu_item.with_icon(variant: ::Icons::GearComponent) %>
+            <% menu_item.with_title(t('marketplace.marketplace.configuration_menu')) %>
+          <% end %>
+        <% end %>
+
+        <%- if policy(marketplace).edit? %>
+          <%= render Marketplace::ManagementMenuItemComponent.new(
+            location: marketplace.location(child: :products)) do |menu_item| %>
+            <% menu_item.with_icon do %>
+              <path stroke-linecap="round" stroke-linejoin="round" d="M9.568 3H5.25A2.25 2.25 0 003 5.25v4.318c0 .597.237 1.17.659 1.591l9.581 9.581c.699.699 1.78.872 2.607.33a18.095 18.095 0 005.223-5.223c.542-.827.369-1.908-.33-2.607L11.16 3.66A2.25 2.25 0 009.568 3z" />
+              <path stroke-linecap="round" stroke-linejoin="round" d="M6 6h.008v.008H6V6z" />
```

Comment:
I would appreciate it if these were named somehow, rather than a blob of svg.

Reply:
If and when they are reused, they can become named components, like Gear. I am not sure naming them for a one-time use makes sense.

Reply:
I like names as a way of revealing intent, since I can't really visualize what this is supposed to look like. Even a little `<!-- price tag icon -->` or something would be appreciated.